### ISSUE TITLE
4kb size regression for Angular hello world app in 5.5.3

### ISSUE THREAD

**Issue by IgorMinar** [ReactiveX/rxjs#3165]
**RxJS version:** 5.5.3

**Code to reproduce:**

https://github.com/angular/angular/blob/master/integration/ng-cli-create.sh

(I know, not the easiest to repro, but I've just heard that this might be a known issue, so let me know if you actually need repro instructions)

**Expected behavior:**

build size @ 5.5.2:
```
-rw-r--r--  1 iminar  eng    1447 Dec  5 11:42 dist/inline.36c6f984ff67507c37b1.bundle.js
-rw-r--r--  1 iminar  eng  154348 Dec  5 11:42 dist/main.7d62bae64a30f7219ca6.bundle.js
-rw-r--r--  1 iminar  eng   61254 Dec  5 11:42 dist/polyfills.7f53b28243f8114c8ce6.bundle.js
```

**Actual behavior:**

build size @ 5.5.3:
```
-rw-r--r--  1 iminar  eng    1447 Dec  5 11:35 dist/inline.455c2227f221ca0aa623.bundle.js
-rw-r--r--  1 iminar  eng  158431 Dec  5 11:35 dist/main.52a3df1e9da61219f4c8.bundle.js
-rw-r--r--  1 iminar  eng   61254 Dec  5 11:35 dist/polyfills.7f53b28243f8114c8ce6.bundle.js
```

The difference is 4.1kb for this hello world app.

**Additional information:**

**Comment by benlesh** [comment]
Thanks for reporting this. It's going to be tricky to debug without a reproduction. I'll try to work with you on this soon.

**Comment by benlesh** [comment]
The two suspects I have are:

https://github.com/ReactiveX/RxJS/commit/d77e3d7

and 

https://github.com/ReactiveX/RxJS/commit/cae5f9b

**Comment by IgorMinar** [comment]
I think if you just do 
```
yarn global add @angular/cli #or npm install -g @angular/cli
ng new foo
cd foo
ng build --prod
```

and then reinstall rxjs, rebuild and compare the sized of files in `dist/*.js`, you'll see the difference

**Comment by benlesh** [comment]
This is resolved with 5.5.4 now... Apparently building with the newer version of TypeScript is bad. LOL.

**Comment by IgorMinar** [comment]
thanks!
